log/log.go

```diff
@@ -30,18 +32,80 @@ var levelPrefix = [...]string{
 	LevelFatal:    "[FATAL] ",
 }
 
+var levelPriority = [...]syslog.Priority{
+	LevelDebug:    syslog.LOG_DEBUG,
+	LevelInfo:     syslog.LOG_INFO,
+	LevelWarning:  syslog.LOG_WARNING,
+	LevelError:    syslog.LOG_ERR,
+	LevelCritical: syslog.LOG_CRIT,
+	LevelFatal:    syslog.LOG_EMERG,
+}
+
 // Level stores the current logging level.
-var Level = LevelDebug
+var Level = LevelInfo
+
+var (
+	// SysLoggers maps each logging level to a *log.Logger that will be used for it.
+	SysLoggers = make([]*golog.Logger, LevelFatal+1)
```

Comment:
Well, each of the `w.Debug(...)`, `w.Warning(...)`, `w.Err(...)` methods sets the severity corresponding to its name, disregarding what the syslogger was constructed with. We use this pattern in Boulder, and the information gets passed along to syslog nicely.

This would take the place of the `golog.New(...)` invocation, which would have the side effect of not double-logging timestamps.